start_server

```diff
@@ -1,3 +1,10 @@
 #!/bin/sh
 
-python emrun.py --no_browser --serve_after_close --serve_after_exit --hostname=0.0.0.0 --port 6932 .
+if command -v python3 &> /dev/null; then
+    python3 emrun.py --no_browser --serve_after_close --serve_after_exit --hostname=0.0.0.0 --port 6932 .
+elif command -v python &> /dev/null; then
+    python emrun.py --no_browser --serve_after_close --serve_after_exit --hostname=0.0.0.0 --port 6932 .
```

Comment:
This change duplicates the parameters onto two lines. Can you refactor so that python interpreter is stored into a variable, and then the cmdline to launch is called via a variable, so that there is no added code duplication?

Reply:
Sure, will do. I actually had the same thought; just didn't get around to implementing it.